influxdb_client/client/write_api_async.py

```diff
@@ -114,12 +115,20 @@ async def write(self, bucket: str, org: str = None,
         self._append_default_tags(record)
 
         payloads = defaultdict(list)
-        self._serialize(record, write_precision, payloads, precision_from_point=False, **kwargs)
-
-        # joint list by \n
-        body = b'\n'.join(payloads[write_precision])
-        response = await self._write_service.post_write_async(org=org, bucket=bucket, body=body,
-                                                              precision=write_precision, async_req=False,
-                                                              _return_http_data_only=False,
-                                                              content_type="text/plain; charset=utf-8")
-        return response[1] in (201, 204)
+        self._serialize(record, write_precision, payloads, precision_from_point=True, **kwargs)
+
+        futures = []
+        for payload in payloads.items():
```

Comment:
I would find more safe and readable to iterate over dict using unpacked item, something like bellow, and then use the key (`payload_precision`) and the value (`payload_lines`) instead of `payload[0]` and `payload[1]`.
Not a strong opinion, though.

```suggestion
        for payload_precision, payload_lines in payloads.items():
```